_inc/lib/core-api/class.jetpack-core-api-module-endpoints.php

```diff
@@ -1069,8 +1069,8 @@ private function _process_onboarding( $data ) {
 
 		// Setup contact page and add a form and/or business info
 		$contact_page = '';
-
-		if ( isset( $data['addContactForm'] ) && $data['addContactForm'] ) {
+		if ( isset( $data['addContactForm'] ) && $data['addContactForm'] &&
+			! get_option( 'jpo_contact_page' ) ) {
```

Comment:
How about keeping this on the same line? I think with the above suggestion the line wouldn't be that long anymore.